autobahn reports/clients/index.html

```diff
@@ -2263,7 +2263,7 @@ <h2>Case 6.4.1</h2>
 Note that PART1 and PART3 are valid UTF-8 in themselves, PART2 is a 0x11000 encoded as in the UTF-8 integer encoding scheme, but the codepoint is invalid (out of range).
 <br><br>MESSAGE PARTS:<br>
 PART1 = κόσμε (cebae1bdb9cf83cebcceb5)<br>
-PART2 = � (f4908080)<br>
+PART2 = ���� (f4908080)<br>

```

Comment:
Not sure what happened here

Reply:
Pretty sure it is the generator and will break when next updated